jquery.confirm.js

```diff
@@ -101,6 +101,13 @@
                     '<h4 class="modal-title">' + settings.title+'</h4>' +
                 '</div>';
         }
+        var cancelButtonHtml = '';
+        if (settings.cancelButton) {
```

Comment:
By default the property will not exist right? So the cancel button will disappear when people upgrade